src/libs/userFeedback.js

```diff
@@ -36,6 +41,7 @@ function shouldBeDisplayed(questionId, hideForDays = 15) {
 }
 
 function rememberAnswer(questionId, answer) {
+  answeredDuringSession.push(questionId);
   localStorage.setItem(
     `${storagePrefix}${questionId}`,
     JSON.stringify({ answer, date: new Date().toISOString() })
```

Comment:
Not related to this PR: using the functions provided by "adapters/store.js" would be safer. They take care of potential errors on the browsers where `localStorage` access is restricted.

Reply:
Indeed, I'll create a new one for that.

Reply:
Hop! https://github.com/Qwant/erdapfel/pull/1119